src/LoggerBase.h

```diff
@@ -391,6 +391,11 @@ class Logger {
      * @brief The logging interval in minutes
      */
     uint16_t _loggingIntervalMinutes = 5;
+    /**
+     * @brief The initial number of samples to log at an interval of 1 minute
+     * for fast field verification
+     */
+    uint8_t _initialShortIntervals = 5;
```

Comment:
There should be a setter function for the initial short intervals. The default should probably be 0 to be consistent with the current code.

Reply:
Will do.